Hoxtygen/home360-server
# ref: refactor get listing enquiries endpoint (HOM-108)

## What does this PR do?
* Refactor endpoint that fetches listing enquiries

## Description of tasks to be completed
* Modify `Service` layer
* Deregister  `WritingConverter`

## How should this be manually tested?
* Clone the repository
* On your PC open a terminal and run git clone repo_url
* Open the `pom.xml` file and download dependencies.
* Run the application.
* Register a user by sending a `POST` request to `http://localhost:8080/api/v1/auth/register` with the required data.
* Open your email, copy the verification link, and extract the token from the link
* Verify your email by sending a `GET` request to `http://localhost:8080/api/v1/auth/verifyEmail?token=${copied_token}`
* Send a `POST` request to `http://localhost:8080/api/v1/auth/login` to login
* Copy the access token from the login response
*  Create a listing by sending a `POST` request to `http://localhost:8080/api/v1/listing` with appropriate data and attach token to header
* Search for a listing and chose one not uploaded by you
* Make an enquiry about the listing with appropriate data and your userId obtained from login response
* Attach the copied token to the request header and send a `GET` request to `http://localhost:8080/api/v1/listing-enquiry`


## What are the relevant Jira board stories?
[HOM-108](https://wasiu-dev.atlassian.net/jira/software/projects/HOM/boards/2?selectedIssue=HOM-108)
Ref #32 



[HOM-108]: https://wasiu-dev.atlassian.net/browse/HOM-108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
